vocabularies/Common.json

```diff
@@ -1356,6 +1356,13 @@
                 "@Common.Experimental": true,
                 "@Core.Description": "Indicates whether the side-effect has already happened",
                 "@Core.LongDescription": "The value of this property typically is a Path expression pointing to a boolean property. It can be used by clients to defer expensive refresh calls until they are actually needed and instead just request the referenced indicator property. Servers can choose to return indicator properties even if not explicitly requested."
+            },
+            "Discretional": {
```

Comment:
Maybe better use [discretionary](https://www.dict.cc/?s=discretionary)?

Unfortunately the JSON is auto-generated from the XML, so you have to change the XML file.